cmd/atlascmd/project.go

```diff
@@ -46,6 +79,9 @@ type Env struct {
 	// Directory containing the migrations for the env.
 	MigrationDir *MigrationDir `spec:"migration_dir"`
 
+	// Values passed from the Env to the schema definition.
+	Values *Values `spec:"values"`
```

Comment:
Why not use the terraform approach and inline these values on the "DefaultExtension"? i.e. without the `values` container. 

https://www.terraform.io/language/modules/develop/composition